src/pages/home/components/tutorial/TutorialSpotlight.tsx

```
@@ -0,0 +1,48 @@
+const TutorialSpotlight = ({ positions }: SpotlightProps) => {
+  const validPositions = positions.filter(
+    (pos): pos is DOMRect => pos !== undefined
+  );
+
+  if (validPositions.length === 0) return null;
+
+  return (
+    <svg
+      className="pointer-events-none absolute inset-0 h-full w-full"
+      style={{ zIndex: 1 }}
+    >
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)